src/utils/helpers.ts

```diff
@@ -105,3 +87,34 @@ export const parseTimeCommitment = (
   const rate = numWeeks > 4 ? hoursPerWeek / 4 : hoursPerWeek;
   return `${rate} hours/${unit} for ${numUnit} ${unit}${plural}`;
 };
+
+// return JS date object with current date at 00:00:00
+export const getCurrentDate = () => {
+  const now = new Date();
+  const nowDate = parseDateAlt(now);
+  return new Date(`${nowDate}T00:00`);
+};
+
+// parse value to three strings depending on truthy, falsy, and nullish
+export const formatTruthy = <
+  T extends string | undefined,
+  F extends string | undefined,
+  N extends string | undefined,
+>(
+  obj: unknown,
+  truthyMessage: T,
+  falsyMessage: F,
+  nullishMessage: N,
+) => {
+  if (obj === null || obj === undefined) return nullishMessage;
+  return obj ? truthyMessage : falsyMessage;
+};
+
+// count number of truthy values
+export const countTruthy = (arr: unknown[]) => {
```

Comment:
forgot to remove that lol
it was intended to check nonempty form fields, but it ended up being more straightforward than I made it up to be

I'll remove it since it's not used.